core/contracts/financial_templates/PricelessPositionManager.sol

```diff
@@ -125,6 +151,8 @@ contract PricelessPositionManager is Testable {
      * @notice Starts a withdrawal request that, if passed, allows the sponsor to withdraw `collateralAmount` from their
      * position. The request will be pending for `withdrawalLiveness`, during which the position can be liquidated.
      */
+    // TODO: should this check if the position is valid first?
+    // TODO: should this check if the contract is pre-expiration?
```

Comment:
isn't this already captured in the require statement?

```
require(
            requestPassTime < expirationTimestamp,
            "Cannot request withdrawal that would pass after contract expires"
        );
```

Reply:
you right.